lib/fluent/plugin/output.rb

```diff
@@ -534,6 +552,35 @@ def handle_stream(tag, es)
         meta_and_data.keys
       end
 
+      def handle_stream_with_standard_format(tag, es)
+        meta_and_data = {}
+        records = 0
```

Comment:
Or, it might be better to use `es.length` if it's larger than 0. Else, do incrementation.